render-test/runner.cpp

```diff
@@ -37,7 +53,8 @@ bool TestRunner::checkImage(mbgl::PremultipliedImage&& actual, TestMetadata& met
 
 #if !TEST_READ_ONLY
     if (getenv("UPDATE")) {
-        mbgl::util::write_file(base + "/expected.png", mbgl::encodePNG(actual));
+        mbgl::filesystem::create_directories(metadata.paths.platformExpectations);
+        mbgl::util::write_file(metadata.paths.platformExpectations.string() + "/expected.png", mbgl::encodePNG(actual));
```

Comment:
What would be the new way of updating `non-platform-specific` expected file?

Reply:
`UPDATE=true yarn run test-render ..` inside `mapbox-gl-js`

Reply:
the latest commit introduces `UPDATE_GENERIC` env variable to update expected results at `mapbox-gl-js` submodule